en/heroku/README.md

```diff
@@ -100,20 +108,13 @@ db_from_env = dj_database_url.config(conn_max_age=500)
 DATABASES['default'].update(db_from_env)
 ```
 
+Note: When you add `whitenoise.middleware.WhiteNoiseMiddleware` to the  `MIDDLEWARE` list make sure it is placed after the django `SecurityMiddlewear`, and leave in the other django middlewear.
+
 It'll do necessary configuration for Heroku.
 
 Then save the file.
 
-## mysite/wsgi.py
-
-Open the `mysite/wsgi.py` file and add these lines at the end:
-
-```python
-from whitenoise.django import DjangoWhiteNoise
-application = DjangoWhiteNoise(application)
-```
-
-All right!
+All right! 
```

Comment:
Why the whitespace change?
```suggestion
All right!
```

Reply:
Good catch, there shouldn't be whitespace there.